_topic_map.yml

```diff
@@ -57,11 +57,13 @@ Topics:
   File: dedicated-networking
 ---
   Name: Understanding ROSA
-  Dir: understanding_rosa
+  Dir: getting_started_rosa
   Distros: openshift-rosa
   Topics:
-  - Name: What is ROSA
-    File: what-is-rosa
+  - Name: About ROSA
+    File: understanding-rosa
+  - Name: Basic Concepts
```

Comment:
Sentence case for headings: "Basic concepts"

Reply:
Done.